robot_skills/src/robot_skills/robot.py

```diff
@@ -16,28 +18,35 @@
 
 CONNECTION_TIMEOUT = 10.0  # Timeout: all ROS connections must be alive within this duration
 
+_ = tf2_geometry_msgs
```

Comment:
What's the point of these?

Reply:
That is also what @reinzor did in `challenge_restaurant`.  I wasn't sure this is actually needed. The most important thing is that is imported I think. Because that registers the conversion functions.